apps/docs/src/components/document/components/component-status-table.tsx

```diff
@@ -33,7 +33,7 @@ export function ComponentStatusPill({
       </div>
       {href ? (
         <IconButton
-          icon={<Icon id="open-in-new" size="sm" />}
+          icon={<Icon icon="open_in_new" size="sm" inline={true} />}
```

Comment:
Will just `inline` work?

Reply:
I put it because the icon works with a `<span>` element while previously was a `<div>`.
Probably, I can still use the `<div>`, but in this way, we can put the icon inline with text.

Reply:
> Will just inline work?
Sorry I just get your question 😬, I have to try

Reply:
Yes, it works.
But in the button it works even without the inline